tools/setupResourceScripts/files/OMOPDataset/sample.json

```diff
@@ -0,0 +1,2 @@
+{"collaborator_participant_id": "77", "collaborator_sample_id": "SM-77", "contamination_rate": 0.001, "exome_gvcf_index_path": "gs://fc-00000000-0000-0000-0000-000000000000/sample.wes.hard-filtered.gvcf.gz.tbi", "exome_gvcf_md5_path": "gs://fc-00000000-0000-0000-0000-000000000000/sample.wes.hard-filtered.gvcf.gz.md5", "exome_gvcf_path": "gs://fc-00000000-0000-0000-0000-000000000000/sample.wes.hard-filtered.gvcf.gz", "genome_crai_path": "gs://fc-00000000-0000-0000-0000-000000000000/sample.cram.crai", "genome_cram_md5_path": "gs://fc-00000000-0000-0000-0000-000000000000/sample.cram.md5", "genome_cram_path": "gs://fc-00000000-0000-0000-0000-000000000000/sample.cram", "mapped_percentage": 99.86, "mean_off_target_coverage": 3.46, "mean_target_coverage": 53.8, "percent_target_bases_at_10x": 98.36, "percent_wgs_bases_at_1x": 94.77, "reblocked_gvcf": "gs://fc-00000000-0000-0000-0000-000000000000/sample.rb.g.vcf.gz", "reblocked_gvcf_index": "gs://fc-00000000-0000-0000-0000-000000000000/sample.rb.g.vcf.gz.tbi", "total_bases": 23772304683}
```

Comment:
this looks like a `jsonl`?

Reply:
Yes, all the data tables are jsonl. I found a number of changes like this that would be an improvement, but decided to limit the work done in this PR, I'll make another tech debt ticket to further clean up this code

Reply:
The file extension is `json`, should it be `jsonl`?

Reply:
I've created https://broadworkbench.atlassian.net/browse/DC-1287 for this follow-up work.